.catkin-lint

```diff
@@ -0,0 +1,2 @@
+[uwrt_mars_rover_drivetrain_hw]
+header_outside_package_include_path = ignore
```

Comment:
I think its relatively safe to have this warning ignored. I think its still good to have the check enabled, but in the times where we do put files in a subfolder in `include` that doesnt match the package name, we should use this file to ignore the check for that package only.